src/uvc-sensor.cpp

```diff
@@ -229,6 +224,12 @@ void uvc_sensor::open( const stream_profiles & requests )
                     // since the content of the OS frame buffer has been copied, it can released ASAP
                     continuation();
 
+                    if (!fh.frame)
+                    {
+                        LOG_INFO("Dropped frame. alloc_frame(...) returned nullptr");
+                        return;
```

Comment:
Maybe better to move the memcpy log to after this block (without moving the time setting itself)?
```
if( diff > 10 )
                        LOG_DEBUG( "!! Frame memcpy took " << diff << " msec" );
```

Otherwise we might see a weird like
```
!! Frame memcpy took " << diff << " msec
Dropped frame. alloc_frame(...) returned nullptr
```